src/lib/modules/Document/DocumentCardGroup/DocumentCardGroup.js

```diff
@@ -44,7 +44,7 @@ class DocumentCardGroup extends Component {
   renderCards() {
     const { data } = this.state;
     if (!data) return null;
-    return data.hits.map(book => {
+    return data.hits.slice(0, this.props.displayNumber).map(book => {
```

Comment:
syntax: the props should be deconstructed, you probably had a warning in VS code for this.

in general it would better if instead of slicing the data hits, you would pass the `displayNumber` prop as a parameter to `fetchDataMethod` which is an API call. By default it return 10, but might be set to more, so we use only a half of the objects in the response we get from the backend. If you pass the page size to the backend, you will  get back exactly the number you need - the response will be smaller and there will be no need to manipulate the output
you can check `withSize` in `borrowingRequest.js` for reference